docs/introduction.rst

```diff
@@ -1,142 +1,131 @@
+  packaging applications rather than focus on their end goals. This delays each
+  distribution's progress.
+- **Old and outdated packages**: LTS distributions often have very old
+  versions of applications packaged natively. This complicates bug
+  reproducibility since applications run in different environments, and
+  developers often have little control over how their applications are packaged
+  by distributions.
+
+Flatpak addresses these issues by enabling developers to distribute
+applications from one source and target the entire Linux desktop.
 
 Reasons to use Flatpak
 ----------------------
 
-Flatpak has some major advantages over most system package managers:
+Flatpak offers major advantages over most system package managers:
 
 - **Universality**: Flatpak allows applications to be installed and run on virtually any Linux
-  distribution. This includes non-GNU distributions, systemd-free distributions,
-  distributions with a read-only operating system (OS), and various architectures without the
-  developer needing the relevant hardware on hand.
-- **Space for innovations**: Flatpak facilitates distribution maintainers to focus on their goals
-  to innovate their distribution.
-- **Stability**: breakage in a Flatpak application will not risk the system from breaking.
-  This is because Flatpak applications and runtimes are contained to not interfere
-  with the system altogether.
-- **Rootless install**: elevated privileges are not required when installing a Flatpak
-  application or a runtime.
-- **Sandboxed applications**: one of Flatpak's main goals is to increase the security of desktop
-  systems by isolating applications from one another. This is achieved using sandboxing and means
-  that, by default, applications that are run with Flatpak have limited access to the host environment.
-
-Flatpak has some major advantages over other universal approaches to distributing
-applications on Linux:
+  distribution, including non-GNU distributions, systemd-free distributions,
+  distributions with a read-only operating system (OS), and various architectures without requiring
+  the developer to have access to the relevant hardware.
+- **Space for innovation**: Flatpak allows distribution maintainers to focus on
+  innovating their distribution without being burdened by packaging concerns.
+- **Stability**: breakages in Flatpak applications won't affect the system,
+  since they run in isolated environments.
+- **Rootless install**: Flatpak does not require elevated privileges to install
+  applications or runtimes.
+- **Sandboxed applications**: Flatpak increases security, one of its main goals, by
+  isolating applications from one another and limiting their access to the host environment.
+
+Flatpak also offers advantages over other universal approaches to Linux application distribution:
 
 - **Decentralized by design**: while Flatpak does provide a centralized service for distributing
```

Comment:
the first sentence is wrong, there is no "central" remote or repo. Change this to `Flatpak offers decentralized hosting and distribution ...`